src/familybot/lib/plugin_admin_actions.py

```
@@ -482,7 +497,9 @@ async def force_new_game_action() -> Dict[str, Any]:
 
 
 async def _collect_wishlists(
-    current_family_members: dict, force_fresh: bool
+    current_family_members: dict,
+    force_fresh: bool,
```

Ruff (0.15.1)

[error] Boolean-typed positional argument in function definition

(FBT001)